impactlab_tools/os/config.py

```diff
@@ -0,0 +1,24 @@
+import sys
+import yaml
+
+def getConfigDictFromFile(filePath):
+    with open(filePath, 'r') as f:
+        confDict = yaml.load(f)
+    return confDict
```

Comment:
Only those that import this module, but sure.  I've made this loading on-demand.